green/test/test_runner.py

```diff
@@ -413,12 +413,13 @@ def testPasses(self):
 
     def test_run_coverage(self):
         """
-        Running coverage in process mode doesn't crash
+        Running coverage in process mode doesn't crash.
         """
         try:
             import coverage
         except ImportError:
             self.skipTest("Coverage needs to be installed for this test")
+        self.assertTrue(coverage.version_info)
```

Comment:
This double up as a way to 'use' coverage so linters do not complained about unused imports.